app/mailers/notifier.rb

```diff
@@ -77,15 +86,21 @@ def order_detail_status_change(order_detail, old_status, new_status, to)
 
   private
 
-  def attach_ical(order)
+  def attach_each_order_ical(order)
     order.order_details.map(&:reservation).compact.each do |reservation|
-      calendar = ReservationCalendar.new(reservation)
-      attachments[calendar.filename] = {
-        mime_type: "text/calendar", content: [calendar.to_ical]
-      }
+      attach_reservation_ical(reservation)
     end
   end
 
+  def attach_reservation_ical(reservation)
+    return if reservation.blank?
```

Comment:
the way the logic is set up is confusing, I'll update it--the extraction only happened as a consequence of needing to attach only the ical stuff for a specific product though